_data/merchants.yml

```diff
@@ -175,4 +175,6 @@
     - name: MoneroDice
       url: https://monerodice.net
     - name: SafeDice
-      url: https://safedice.com
+      url: https://safedice.comet
```

Comment:
`.comet` should be `.com` :smile:

Reply:
Thanks. I blame lack of coffee and Hurricane Irma.

Reply:
At least you didn't get hit by a comet.